### ISSUE TITLE
children for abnormalFunctionalityOfCellularComponent

### ISSUE THREAD

**Issue by pfey03** [obophenotype/upheno#755]
I have phenotypes decreased and increased F-actin polymerization at leading edge. Many annotations.

From the example 'synaptic transmission at the neuromuscular junction"

 A GO process and cellular component like our phenotypes.

abnormallyDecresedFunctionalityOfCellularComponent
abnormallyIncresedFunctionalityOfCellularComponent

@matentzn Let me know if I should add these patterns

**Comment by matentzn** [comment]
Yes please! Based on https://raw.githubusercontent.com/obophenotype/upheno/master/src/patterns/dosdp-dev/abnormalFunctionalityOfCellularComponent.yaml

Thank you @pfey03

**Comment by pfey03** [comment]
ok will do and hope this comma doesn't sneak in again

**Comment by dosumis** [comment]
Isn't this better expressed as an abnormal process that occurs in X or that has participant X?  We have lots of axiomatization in GO about such processes?
